packages/create-sitecore-jss/src/templates/nextjs/src/lib/page-props.ts

```diff
@@ -13,4 +14,5 @@ export type SitecorePageProps = {
   componentProps: ComponentPropsCollection;
   notFound: boolean;
   layoutData: LayoutServiceData;
+  errorHandlingPages?: ErrorHandlingType;
```

Comment:
If it's not used in the base app, it shouldn't be here, it's SXA related property